regression/goto-analyzer-simplify/simplify-complex-expression/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--variable
+r == 2
+^SIGNAL=0$
+^EXIT=6$
```

Comment:
Why does it exit with this "error" exit code?

Reply:
It's not an error. It's something goto-analyzer does in normal operation for some reason (goto_analyzer_parse_options.cpp:283, coming from goto_analyzer_parse_options.cpp:636). There's no doc comment or anything like that explaining it as far as I can tell.

Reply:
Check the corresponding lines in develop.  I fixed a number of return code errors when merging goto-analyze-6 to develop.  Thus 6 would be right on this branch but will need to be fixed when we get to the merge of the variable sensitivity branch into develop.

Reply:
Thanks both. Definitively out of scope for this PR, but weird nonetheless as it will make the tool really hard to use in a non-interactive setting.

Reply:
#fixedindevelop ...